platform/admin/core/admin-core-appservice/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -60,9 +61,17 @@
     <bean id="restSvc" class="org.codice.ddf.admin.application.rest.ApplicationUploadEndpoint">
         <argument ref="appService"/>
     </bean>
-    
+
     <reference-list id="applicationPluginList"
                     interface="org.codice.ddf.admin.application.plugin.ApplicationPlugin"
                     availability="optional"/>
 
+    <bean id="profileMigratable"
+          class="org.codice.ddf.admin.application.service.migratable.ProfileMigratable">
+        <argument ref="appService"/>
+        <argument ref="featuresService"/>
+    </bean>
+
+    <service id="profileMigratableService" ref="profileMigratable"
```

Comment:
❓ Should we add a comment explaining why `ranking` is set to `9999`?

Reply:
Although documented in the migration api doc, I will add a comment.